src/Tools/ExternalAccess/FSharp/Internal/Completion/FSharpInternalCommonCompletionProvider.cs

```diff
@@ -34,5 +35,11 @@ internal override bool IsInsertionTrigger(SourceText text, int insertedCharacter
         {
             return _provider.IsInsertionTrigger(text, insertedCharacterPosition, options);
         }
+
+        /// <summary>
+        /// Pulled from https://github.com/dotnet/fsharp/blob/master/vsintegration/src/FSharp.Editor/Completion/FileSystemCompletion.fs#L140
+        /// but this is incredibly fragile.
+        /// </summary>
+        internal override ImmutableHashSet<char> PossibleTriggerCharacters => ImmutableHashSet.Create('"', '\\', ',', '/');
```

Comment:
indeed.  let's notify them of this so tehy can take ownership of this if necessary.